releasenotes/notes/weights-dae0759ea750cffd.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added two new properties to the `TrainableModel` class:
+      - ``fit_result`` returns a resulting object from the optimization procedure. Please refers to
```

Comment:
We could make Trainable model a link, or perhaps instead both the `fit-result` an `weights` be links to the new properties themselves so that a reader of the release note can go direct there.

Reply:
Added links.